README.md

```diff
@@ -1 +1,99 @@
-Coming soon…
+# UI Component Best Practices <!-- omit in toc -->
+
+A bunch of best practices to help achieve User Interface (UI) components, or
+presentational components, that are:
+
+- accessible
+- consumable
+- portable
+- reusable
+- and robust.
+
+## Table of Contents <!-- omit in toc -->
+
+- [Caveats](#caveats)
+- [The tl;dr Version](#the-tldr-version)
+- [Components Cannot Apply External Spacing](#components-cannot-apply-external-spacing)
```

Comment:
`should not` might be a better way of phrasing this, as there is an exceptions section